utils/fdb_wrapper_cpp/src/fdbcs.cpp

```diff
@@ -111,6 +121,13 @@ bool Transaction::commit() const
       std::cerr << "fdb_future_block_until_ready error, code: " << (int)err << std::endl;
       return false;
     }
+    err = fdb_future_get_error(future);
```

Comment:
Could you ellaborate on this sequence you added? A programmer is obliged to always read an error?

Reply:
If the future has an error, we need to handle it/